app/src/main/java/fr/free/nrw/commons/achievements/AchievementsActivity.java

```diff
@@ -316,8 +316,8 @@ private void setUploadProgress(int uploadCount){
 
     private void setZeroAchievements() {
         AlertDialog.Builder builder=new AlertDialog.Builder(this)
-                .setMessage("You haven't made any contributions yet")
-                .setPositiveButton("Ok", (dialog, which) -> {
+                .setMessage(getString(R.string.achiev_activity_set_zero_achiev))
+                .setPositiveButton(getString(R.string.ok), (dialog, which) -> {
```

Comment:
Similarly this could be not_achievements_yet

Reply:
I am sorry I meant, "not_achievements_yet" :)

Reply:
I would say no_achievements_yet is grammatically better? :-)

Reply:
Yes, my bad, the same typo again, @PavelAplevich  would be great if you could make it "no_achievements_yet"